kebechet/utils.py

```diff
@@ -30,6 +30,18 @@
 
 _LOGGER = logging.getLogger(__name__)
 
+# NOTE: does not support other branches
+services = {
```

Comment:
From coding design point of view, it will make more sense to create a class for this and provide methods. The overall manipulation will be much straightforward and the class with methods (all can be `classmethods`) will appear in generated docs as well.